org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/TraceValue.java

```diff
@@ -23,7 +23,7 @@ private TraceValue() {
 
 	public static final String Off = "off";
 
-	public static final String Message = "message";
+	public static final String Message = "messages";
```

Comment:
Should this field be changed to "Messages" too for consistency with the surrounding ones?

Reply:
Probably. I was just trying to avoid API breaking.

Reply:
Should not we then deprecate the old constant and introduce a new one instead? Or, assuming no one has actually used it (given that no issue has been reported about it so far), could not we just go ahead and change both the constant name and value (with a corresponding note about the breaking change)? Anyway, changing the published constant value is already a breaking change, formally.

Reply:
Since it was actually a typo in the spec, I will go with deprecation.